gradio/components/code.py

```diff
@@ -155,20 +154,16 @@ def preprocess(self, payload: str | None) -> str | None:
         """
         return payload
 
-    def postprocess(self, value: tuple[str] | str | None) -> None | str:
+    def postprocess(self, value: str | None) -> None | str:
         """
         Parameters:
-            value: Expects a `str` of code or a single-element `tuple`: (filepath,) with the `str` path to a file containing the code.
+            value: Expects a `str` of code.
         Returns:
             Returns the code as a `str`.
         """
         if value is None:
```

Comment:
If someone does pass in a tuple, can we raise an informative exception telling them this is no longer supported in `gr.Code`?